pkg/kubelet/managed/managed.go

```diff
@@ -25,13 +26,27 @@ import (
 )
 
 var (
-	pinnedManagementEnabled      bool
-	pinnedManagementFilename     = "/etc/kubernetes/openshift-workload-pinning"
-	WorkloadsAnnotationPrefix    = "workload.openshift.io/"
-	WorkloadsCapacitySuffix      = "workload.openshift.io/cores"
-	ContainerCpuAnnotationFormat = "io.openshift.workload.%v.cpushares/%v"
+	pinnedManagementEnabled   bool
+	pinnedManagementFilename  = "/etc/kubernetes/openshift-workload-pinning"
+	WorkloadsAnnotationPrefix = "workload.openshift.io/"
```

Comment:
This has been changed to target.workload.openshift.io. See 
 https://github.com/openshift/enhancements/blob/master/enhancements/management-workload-partitioning.md

Reply:
Thank you. Fixed.